### PR TITLE
README.md: add TiDB Community Architecture link

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: governance/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Special Interest Group(SIG)
+
+**Special Interest Groups (SIGs)** are persistent open groups that focus on a
+module of TiDB. SIGs must have open and transparent proceedings. The purpose of
+a SIG is to own and develop a module of TiDB.
+
+#### How to create a SIG(WIP)?
+
+#### SIG Governance(WIP)
```
Comment (gingerkidney) [inline governance/README.md]: can we use the same charter with TiKV?

Comment (zz-jason) [reply governance/README.md]: any suggestion?

File: README.md [hunk 2]
```diff
@@ -38,10 +40,8 @@ For more specific topics, join
 TiDB has the following types of groups that are officially supported:
```
Comment (dcalvin) [inline README.md]: Do we need this section here?

Comment (zz-jason) [reply README.md]: how about removing them after adding the governance of SIG and WGs?

Comment (dcalvin) [reply README.md]: OK

Comment (dcalvin) [reply README.md]: I think we can leave this section and add the link to the governance file when it is merged.

Comment (zz-jason) [reply README.md]: Do you mean to remove the `Governance` section in this file?

Comment (gingerkidney) [reply README.md]: I think Calvin means that we can replace the governance section with a link. So we do not need to maintain 2 files.

Comment (dcalvin) [reply README.md]: Sorry about the confusion. You can refer to TiKV community readme. We can have the section here, but only provides the link to Governance.

Comment (dcalvin) [reply README.md]: Correct.

Comment (zz-jason) [reply README.md]: But we don't have this link in the current repo 😂

Comment (dcalvin) [reply README.md]: We can remove the link text and add it after the governance is merged.

**Pull request merged**